File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/AutocompleteItem.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class AutocompleteItem(
  val id: String,
  val insertText: String,
  val range: Range,
)

File: agent/bindings/csharp/AutocompleteItem.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class AutocompleteItem
  {
    [JsonProperty(PropertyName = "id")]
    public string Id { get; set; }
    [JsonProperty(PropertyName = "insertText")]
    public string InsertText { get; set; }
    [JsonProperty(PropertyName = "range")]
    public Range Range { get; set; }
  }
}
